packages/@react-spectrum/table/src/table.css

```diff
@@ -49,3 +64,148 @@
+  box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
+}
+
+.react-spectrum-Table-InsertionIndicator {
+  z-index: 3;
+  width: calc(100% - (2 * var(--spectrum-table-dropindicator-circle-size)));
+  inset-inline-start: var(--spectrum-table-dropindicator-circle-size);
+  position: absolute;
+  outline: none;
+
+  &.react-spectrum-Table-InsertionIndicator--dropTarget {
+    background: var(--spectrum-table-dropindicator-border-color);
+    border-bottom: 2px solid var(--spectrum-table-dropindicator-border-color);
+
+    /* forced-color-adjust: none, so that background color
+    will render border for insertion indicator. */
+    forced-color-adjust: none;
+
+    &:before {
+      left: calc(var(--spectrum-table-dropindicator-circle-size) * -1);
+    }
+
+    &:after {
+      right: calc(var(--spectrum-table-dropindicator-circle-size) * -1);
+    }
+
+    &:before,
+    &:after {
+      content: '';
+      position: absolute;
+      top: calc(var(--spectrum-table-dropindicator-circle-size) * -1 / 2 + var(--spectrum-table-dropindicator-border-size) / 2);
+      width: var(--spectrum-table-dropindicator-circle-size);
+      height: var(--spectrum-table-dropindicator-circle-size);
+      border-radius: 50%;
+      border: var(--spectrum-table-dropindicator-border-size) solid;
+      box-sizing: border-box;
+      border-color: var(--spectrum-table-dropindicator-circle-border-color);
+      background-color: var(--spectrum-table-background-color);
+      z-index: 5;
+
+      /* forced-color-adjust: none, so that box-shadow and background-color styles will render */
+      forced-color-adjust: none;
+    }
+  }
+}
+
+.react-spectrum-Table-dragButtonCell {
+  padding: var(--spectrum-global-dimension-size-40);
+}
+
+.spectrum-Table-row-draghandle-button {
+  border-radius: var(--spectrum-alias-border-radius-regular);
+  padding: var(--spectrum-global-dimension-size-10);
+  
+  &:focus-ring {
+    box-shadow: inset 0 0 0 2px var(--spectrum-global-color-blue-400);
+    outline: none;
+
+    /* forced-color-adjust: none, so that box-shadow style will render */
+    forced-color-adjust: none;
+  }
+  svg {
+    margin: auto 0;
+    display: block;
+    padding: var(--spectrum-global-dimension-size-40);
+  }
+}
+
+.react-spectrum-dragButtonHeadCell {
+  padding: 0;
+  height: inherit;
+}
+
+.react-spectrum-Table-row--dropTarget {
+  border: 1px var(--spectrum-table-dropindicator-border-color);
+  box-shadow: inset 2px 0 0 0 var(--spectrum-table-dropindicator-border-color),
+    inset -2px 0 0 0 var(--spectrum-table-dropindicator-border-color),
+    inset 0 -3px 0 0 var(--spectrum-table-dropindicator-border-color),
+    inset 0 2px 0 0 var(--spectrum-table-dropindicator-border-color);
+
+  /* forced-color-adjust: none, so that box-shadow style will render */
+  forced-color-adjust: none;
+
+  /** Needed to style spectrum-Table-cellWrapper */
+  > * {
+    background-color: var(--spectrum-table-droptarget-background-color) !important;
```

Comment:
odd it needs important, why is that? or what value is winning over it?

Reply:
Updated to add logic to the renderWrapper that applies a class to the cell if the row is being dropped on, so this isn't needed anymore.